app/api/orders.py

```diff
@@ -499,3 +500,27 @@ def alipay_return_uri(order_identifier):
             return jsonify(status=False, error='Charge object failure')
     except TypeError:
         return jsonify(status=False, error='Source object status error')
+
+
+@order_misc_routes.route('/orders/<string:order_identifier>/omise-checkout', methods=['POST', 'GET'])
+def omise_checkout(order_identifier):
+    """
+    Charging the user and returning payment response for Omise Gateway
+    :param order_identifier:
+    :return: JSON response of the payment status.
+    """
+    token = request.form.get("omiseToken")
+    try:
+        charge = OmisePaymentsManager.charge_payment(order_identifier, token)
+    except omise.errors.BaseError as e:
+        logging.error(f"""OmiseError: {repr(e)}.  See https://www.omise.co/api-errors""")
```

Comment:
please make the syntax python 3.6 compatible

Reply:
Why?

Reply:
Upgrade your python please